db/migrate/20190509014833_create_receivers.rb

```diff
@@ -1,13 +1,17 @@
+# Receivers is a temporary table.  When we get the data from Highlands SSO we'll remove the receivers table.
 class CreateReceivers < ActiveRecord::Migration[6.0]
   def change
     create_table :receivers do |t|
+      t.string :receiver # Highlands SSO ID
```

Comment:
I know in the spec in the post request we should have a `receiver`, but here in the model can we have `receiver_sso_id`? With this, anyone who read it, it will understand where it will be used and where this come from(SSO).

Reply:
@knewter  ^